### PR TITLE
Fix passing of File.cp_r! on_conflict option

### PR DESCRIPTION
**Pull request by michallepicki**
found by [dialyzer](https://github.com/michallepicki/elixir-lang-dialyzer-runs/runs/7531330911?check_suite_focus=true#step%3A7%3A791=) :)

### CODE REVIEW

File: lib/mix/lib/mix/release.ex [hunk 1]
```diff
@@ -773,7 +773,7 @@ defmodule Mix.Release do
 
     release.erts_source
     |> Path.join("bin")
-    |> File.cp_r!(destination, fn _, _ -> false end)
+    |> File.cp_r!(destination, on_conflict: fn _, _ -> false end)
```
Comment (michallepicki) [inline lib/mix/lib/mix/release.ex:776]:
```suggestion
    |> File.cp_r!(destination)
```

File: lib/mix/lib/mix/release.ex [hunk 2]
```diff
@@ -773,7 +773,7 @@ defmodule Mix.Release do
 
     release.erts_source
     |> Path.join("bin")
-    |> File.cp_r!(destination, fn _, _ -> false end)
+    |> File.cp_r!(destination)
```
Comment (josevalim) [inline lib/mix/lib/mix/release.ex:776]:
```suggestion
    |> File.cp_r!(destination, on_conflict: fn _, _ -> false end)
```

Comment (michallepicki) [reply lib/mix/lib/mix/release.ex]: Ah, false vs true, my bad!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**